pytorch_impl/datareader.py

```
@@ -0,0 +1,403 @@
+            else:
+                return np.vectorize(self.__map_haps_2_ind)(data.values.T)
+        else:
+            return data.replace(self.replacement_dict).values.T.astype(np.int32)
+
+    def get_ref_set(self, starting_var_index=0, ending_var_index=0) -> np.ndarray:
+        if 0 <= starting_var_index < ending_var_index:
+            return self.__get_forward_data(
+                data=self.reference_panel.iloc[starting_var_index:ending_var_index, self.ref_sample_value_index - 1:])
+        else:
+            pprint("No variant indices provided or indices not valid, using the whole sequence...")
+            return self.__get_forward_data(data=self.reference_panel.iloc[:, self.ref_sample_value_index - 1:])
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)